scrapers/sis_scraper/main.py

```diff
@@ -76,7 +76,7 @@ async def get_section_information(section_url):
         for seat_type in seating[1:]:
             kind = seat_type.find("th").text
             capacity, actual, remaining = tuple(
-                int(x.text) for x in seat_type.findAll("td")
+                int(x.text.replace("\xa0", "0")) for x in seat_type.findAll("td")
```

Comment:
I'd rather fix the root cause of the incorrect encoding rather than patch the issue after the fact.

Reply:
Unfortunately I checked in SIS itself, and the nonbreaking space (which is the character replaced here) is in the HTML itself and isn't an encoding error, and there isn't really anything I can do about it.

In some old course data, waitlist and crosslists seating data which are empty and normally would be [ 0, 0, 0 ] are instead [ u'\xa0', u'\xa0', u'\xa0' ] (there is not even a zero there; even if it was a normal space instead of a nonbreaking space, the scraper still crashes), and this was the only way I could fix it. You can see the problem in this screenshot (term = 199805):

![image](https://user-images.githubusercontent.com/116031952/219815420-1c362f4b-8689-4b8e-bdd2-2f8dbc1cc335.png)

Reply:
SIS moment